server/src/main/java/com/datastrato/gravitino/server/web/ObjectMapperProvider.java

```diff
@@ -4,20 +4,46 @@
  */
 package com.datastrato.gravitino.server.web;
 
-import com.datastrato.gravitino.json.JsonUtils;
 import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.MapperFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.databind.cfg.EnumFeature;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
 import javax.ws.rs.ext.ContextResolver;
 import javax.ws.rs.ext.Provider;
 
 @Provider
 public class ObjectMapperProvider implements ContextResolver<ObjectMapper> {
 
+  private static class ObjectMapperHolder {
+    private static final ObjectMapper INSTANCE =
+        JsonMapper.builder()
+            .configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false)
+            .configure(EnumFeature.WRITE_ENUMS_TO_LOWERCASE, true)
+            .enable(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS)
+            .build()
+            .setSerializationInclusion(JsonInclude.Include.NON_NULL)
+            .registerModule(new JavaTimeModule());
+  }
+
+  /**
+   * Retrieves a globally shared {@link ObjectMapper} instance.
+   *
+   * <p>Note: This ObjectMapper is shared across all users. If you need to modify the default
```

Comment:
`shared across all users`, I think it is not accurate for server-side object mapper, you need to improve the comment here.

Reply:
done